public/pages/workflows/new_workflow/utils.ts

```diff
@@ -59,6 +62,39 @@ export function enrichPresetWorkflowWithUiMetadata(
     }
   }
 
+  if (semver.gte(version, '2.17.0') && semver.lt(version, '2.19.0')) {
```

Comment:
let's use the 2.17 constant here instead of hardcoding, and please create one for 2.19.0 as well.

Reply:
Replace all the hardcoded string with the 2.17 and 2.19 constants in this util file, processor_list file and new_workflow file